tools/ddrgen/src/scanners/dwarf/AixDwarfParser.cpp

```diff
@@ -280,14 +248,13 @@ static int
 parseDwarfDie(const string line)
 {
 	int ret = DW_DLV_OK;
+	double typeID;
 	/* get the type of the data of the line being parsed */
-	Parsed_Declaration_Data *parsedDeclarationData = new Parsed_Declaration_Data();
-	Dwarf_Half tag = parseDeclarationLine(line, parsedDeclarationData);
+	Dwarf_Half tag = parseDeclarationLine(line, typeID);
```

Comment:
Since our coding standard does not allow reference types, consider the equivalent without reference type arguments: Dwarf_Half tag = parseDeclarationLine(line, &typeID);